deptrac.yaml

```diff
@@ -137,6 +137,7 @@ parameters:
       - Resolver
     Dependencies:
       - Utils
+      - AstRunner
```

Comment:
We now have a cyclic dependency. Effectively this was already the case when including DependencyEmitter and considering the ignored violations in the baseline. I think the appropriate fix would be to have both AstRunner + Dependencies in a shared layer, either by changing this config file or changing the namespace. Renaming the EmitterInterface and moving the emitters already is a big change, so I will create a separate issue for this.

Reply:
You can already see it from the look of the depfile itself, but my thinking is if we want to use deptrac to define DDD domains, we need to reshuffle a lot of classes from one directory to a different one for it to make sense.

In the ideal world, (and this is what I am doing in my project). I have one layer/domain per directory. If it is big, I have it as a group and do the same thing one directory down (again layer per directory). If you are fine with this approach, I am happy to do this reorganization of files.